girder/api/v1/system.py

```diff
@@ -149,8 +150,13 @@ def getPlugins(self, params):
         Return the plugin information for the system. This includes a list of
         all of the currently enabled plugins, as well as
         """
+        allPlugins = plugin_utilities.findAllPlugins()
+        for plugin in allPlugins.values():
+            if 'dependencies' in plugin:
+                plugin['dependencies'] = list(plugin['dependencies'])
```

Comment:
@danlamanna do you mind if I add a commit on this branch to make it so the conversion of sets to lists is done automatically in our REST layer? Then I will remove this change.